rasa/core/actions/action.py

```diff
@@ -738,8 +744,16 @@ async def run(
             logger.debug(
                 "Calling action endpoint to run action '{}'.".format(self.name())
             )
+
+            should_compress = get_bool_env_variable(
+                COMPRESS_ACTION_SERVER_REQUEST_ENV_NAME,
+                DEFAULT_COMPRESS_ACTION_SERVER_REQUEST,
+            )
```

Comment:
We need to handle the potentially raised `RasaException` recommended above here.

Reply:
I don't think we need to as this should raise? cf. line 779 that also raises a `RasaException`